src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -791,11 +807,12 @@ public TypeHandle GetArrayElementTypeHandle()
     }
 
     // Subset of src\vm\methodtable.h
-    [StructLayout(LayoutKind.Explicit)]
+    [StructLayout(LayoutKind.Sequential)]
     internal unsafe struct MethodTableAuxiliaryData
     {
-        [FieldOffset(0)]
         private uint Flags;
+        private void* LoaderModule;
+        private nint ExposedClassObjectRaw;
```

Comment:
What prevents it being a regular `RuntimeType?` field? Structure layout of managed field?

Reply:
Note this is a projection of a native type and must be blittable in the managed sense.